### ISSUE TITLE
Add support for gifv media attachments

### ISSUE THREAD

**Issue by CaptainSpam** [CaptainSpam/mastowidget#1]
Ideally, anything that comes in via the gifv type can just be treated like an animated gif; that is, one img element oughta do it, just like how static images are done.  However, I'd like to set them such that they don't necessarily animate until the user clicks on them, which might require more fiddling, a config option in the JS, and maybe some way to display metadata (i.e. how long the animation is).

**Comment by CaptainSpam** [comment]
On closer inspection of the API, Mastodon considers a "gifv" to be a "[looping, soundless animation](https://docs.joinmastodon.org/entities/MediaAttachment/#type)".  I'm going to need to experiment a bit to figure out what they mean by that; the example used in the docs links to an MP4, not a GIF, and there doesn't appear to be an option when uploading a video media attachment to treat it as looping, which makes me wonder if the server does that check on upload, automatically assigning the "gifv" type to any video that has no audio track.  Similarly, I should make sure I know how it's treating an animated GIF (I have my doubts Mastodon would have special handling for aPNG, and nobody outside of the PNG Development Group has ever liked MNG).

Long story short, this is now sounding like I should handle attachments of type gifv as _videos_, not _images_, and just set the loop attribute on them.  But before that, I'll need to concoct a test video.

**Comment by CaptainSpam** [comment]
There we go.  gifv support is now live.
